apps/opik-backend/src/main/java/com/comet/opik/domain/ExperimentItemService.java

```diff
@@ -126,7 +126,7 @@ public Flux<ExperimentItem> getExperimentItems(@NonNull ExperimentItemStreamRequ
                 .subscribeOn(Schedulers.boundedElastic())
                 .collect(Collectors.mapping(Experiment::id, Collectors.toUnmodifiableSet()))
                 .flatMapMany(experimentIds -> experimentItemDAO.getItems(
-                        experimentIds, request.limit(), request.lastRetrievedId()));
+                        experimentIds, request));
```

Comment:
It's true that we use the same class as API and model for the domain objects. But I would avoid cascading down a "Request" object; maybe creating a criteria object and mapping it would be better.